scripts/addr2line.py

```diff
@@ -83,41 +84,148 @@ def __call__(self, address):
         self._input.stdin.flush()
         return self._read_resolved_address()
 
+class KernelResolver:
+    """A reserver for kernel addresses which tries to read from /proc/kallsyms."""
+
+    LAST_SYMBOL_MAX_SIZE = 1024
+
+    def __init__(self, binary, *args, **kwargs):
+        assert binary == '<kernel>', 'this resolver only makes sense for the kernel'
```

Comment:
Maybe define '<kernel>' as a constant rather than repeating the string literal?

Reply:
thanks fixed in next (and this particular use was removed since I fixed the call site not to pass module when creating a kernel resolver